docs/source/installation/windows.rst

```diff
@@ -0,0 +1,43 @@
+********************
+Installation on Windows
+********************
+
+Python package
+==============
+
+1. `Install Python <https://www.python.org/downloads>`_ version 2.7.12, 2.8 or 3.4 upwards - Python 2.7.12 onwards, contains pip in the installation package.
```

Comment:
@vincpa Screenshot? There are exactly no mentions of `2.8` anywhere in the repository, except for seven floating-point constants in tests/test_segments.py (like `62.857142857142854`).